docs/output.md

```diff
@@ -6,13 +6,16 @@ This document describes the output produced by the pipeline. Most of the plots a
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
+TODO: Should here be added which output is cDNA/DNA specific?
```

Comment:
I think that makes sense, to put CDNA-ONLY, DNA-ONLY, or SHARED, or something to that effect in the sections and maybe put a blurb at the top that there are notes about which steps are type-specific.